Test/DefaultValues/CreateTableDefaults.hs

```diff
@@ -0,0 +1,84 @@
+{-# OPTIONS_GHC -Wno-ambiguous-fields #-}
+module Test.DefaultValues.CreateTableDefaults where
```

Comment:
As this is used outside the testsuite scope (e.g. in `IHP/IDE/SchemaDesigner/SchemaOperations`) shouldn't we move this to something like `IHP.IDE.SchemaDesigner.Defaults`?

Reply:
That's understandable, I was unsure where best to place them as I was focused more on getting started on making the defaults. I'll make a note to move and rename as needed.

Reply:
That's understandable, I was unsure where best to place them as I was focused more on getting started on making the defaults. I'll make a note to move and rename as needed.